### ISSUE TITLE
Add interoperable output activations for all models

### ISSUE THREAD

**Issue by DavidLandup0** [DavidLandup0/deepvision#25]
- None (output logits or for regression tasks)
- Callable function (sigmoid, softmax, etc.)
